### PR TITLE
Rbta119-bugreport-statusbadges-properties-boytsova

### PR DESCRIPTION
**Pull request by Foryoushka**
Created tests for  bugreport-statusbadges-properties. Also created helpers, added expected and selectors for these tests.

### CODE REVIEW

File: test/expected/bug-report.js [hunk 1]
```diff
@@ -35,6 +35,17 @@ export default {
...
+  badgeOpenColor: '#007bff',
+  badgeCannotReproduceFontColor: '#212529',
+  badgeClosedColor: '#343a40',
+  badgeDuplicateColor: '#343a40',
+  badgeProgressColor: '#28a745',
+  badgeVerificationColor: '#17a2b8',
+  badgeReopenedColor: '#dc3545',
+  badgeCannotReproduceColor: '#ffc107',
+  badgeWontFixColor: '#6c757d',
+
```
Comment (KharkovaIrina) [inline test/expected/bug-report.js]: two extra space lines

Comment (Foryoushka) [reply test/expected/bug-report.js]: Going to make another PR, and will put these small changes there, maybe this one could be merged earlier

Comment (Foryoushka) [reply test/expected/bug-report.js]: Thank you for the approve

**Comment by Foryoushka** [comment]
closed as I created a new one with more tests

**Pull request closed without merging**